department-of-veterans-affairs/va.gov-cms
# VACMS-13017: tools updated to use npm

## Description

Closes #13017.

## Testing done
Tested locally. Ran the composer scripts `va:theme:compile`, `va:ds:drupal` and `va:ds:storybook`.

## Screenshots


## QA steps

What needs to be checked to prove this works?  
What needs to be checked to prove it didn't break any related things?  
What variations of circumstances (users, actions, values) need to be checked?  

As user _uid_ with _user_role_
1. Check the tugboat logs and verify that everything builds correctly
   - [ ] Validate that storybook is running
   - [ ] Validate that the theme compiles correctly in the logs.


### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing annoucement

Is an announcement needed to let editors know of this change?
